velox/exec/Driver.h

```diff
@@ -128,7 +128,9 @@ enum class BlockingReason {
   kNotBlocked,
   kWaitForConsumer,
   kWaitForSplit,
-  kWaitForExchange,
+  /// Some operators can get blocked due to the producer(s) they are waiting
```

Comment:
Perhaps, clarify that this applies only to source operators and is used by LocalExchange, LocalMergeExchange, Exchange and MergeExchange.